src/common/dateFormat.js

```diff
@@ -0,0 +1,12 @@
+export const changeDateFormat = date => {
+    const splitDate = date.split('-');
+    if (splitDate.count === 0) {
+        return null;
+    }
+    
+    const year = splitDate[0];
+    const month = splitDate[1];
+    const day = splitDate[2];
+
+    return month + '-' + day + '-' + year;
+}
```

Comment:
I think this function could be replaced with `moment('1996-01-22', 'YYYY-MM-DD').format('MM-DD-YYYY')`